routers/api/v1/api.go

```diff
@@ -691,9 +691,9 @@ func RegisterRoutes(m *macaron.Macaron) {
 							m.Post("/stop", reqToken(), repo.StopIssueStopwatch)
 						})
 						m.Group("/subscriptions", func() {
-							m.Get("", bind(api.User{}), repo.GetIssueSubscribers)
-							m.Put("/:user", repo.AddIssueSubscription)
-							m.Delete("/:user", repo.DelIssueSubscription)
+							m.Get("", repo.GetIssueSubscribers)
```

Comment:
Why doesn't this need `reqToken()`?

In fact, I don't know if this bug comes from this PR, but I've just got some users' emails using this endpoint without presenting any credentials!

Reply:
@guillep2k 
It set/unset subscribtion - and currently the only check is ctx.user == user || ctx.user.is_admin
with token req it should be more secure ?

Reply:
@guillep2k and m.Get("/subscriptions", user.GetWatchedRepos) also dont require Token ...

can i resolve this?

Reply:
@guillep2k should I add `reqToken()` ?!?

or can I "resolve" this for now?!?